nbconvert/nbconvertapp.py

```diff
@@ -170,12 +177,13 @@ def _classes_default(self):
         """.format(get_export_names()))
 
     # Writer specific variables
-    writer = Instance('nbconvert.writers.base.WriterBase',  
-                      help="""Instance of the writer class used to write the 
+    writer = Instance('nbconvert.writers.base.WriterBase',
+                      help="""Instance of the writer class used to write the
                       results of the conversion.""", allow_none=True)
-    writer_class = DottedObjectName('FilesWriter', config=True, 
-                                    help="""Writer class used to write the 
+    writer_class = DottedObjectName('FilesWriter', config=True,
+                                    help="""Writer class used to write the
                                     results of the conversion""")
+    from_stdin   = Bool(False, config=True, help="read a single notebook from stdin.")
```

Comment:
This is located in a confusing place in the file: it's definitely not a writer-specific variable. I suggest to move to the group of `# Other configurable variables`.